meteor/server/api/rest/rest.ts

```diff
@@ -37,16 +37,13 @@ function typeConvertUrlParameters(args: any[]) {
 			if (!_.isNaN(Number(val))) {
 				val = Number(val)
 			} else {
-				let json: any = null
 				try {
-					json = JSON.parse(val)
+					val = JSON.parse(val)
 				} catch (e) {
 					// ignore
```

Comment:
How common is this ignore path hit?
If very rare, maybe a debug log statement, indicating the error could be of use in the future.

Reply:
I guess this only happens when we failed to parse the JSON, in which case we continue and refer to the handling of the call itself later. I'm not sure what the benefit of adding a log here would be.